batchrun/tests/test_models.py

```diff
@@ -0,0 +1,42 @@
+from datetime import timedelta
+
+import pytest
+from django.core.exceptions import ObjectDoesNotExist
+from django.utils import timezone
+
+from batchrun.models import JobRun
+
+
+@pytest.mark.django_db
+def test_jobrunlog_delete(job_run_log_factory):
+    jobrunlog_entry_dict = job_run_log_factory(
+        start=timezone.now() - timedelta(hours=1),
+        end=timezone.now(),
+        entry_count=100,
+        error_count=1,
+        content="test content",
+        entry_data={"test": "data"},
+    )
+
+    jobrun_dict: JobRun = JobRun.objects.get(pk=jobrunlog_entry_dict.run.pk)
+    assert jobrun_dict.log is not None
+    jobrun_dict.delete_logs()
+    jobrun_dict.refresh_from_db()
+    with pytest.raises(ObjectDoesNotExist):
+        jobrun_dict.log
+
```

Comment:
Could the test be split to two parts: one with a dict-type log entry, and another with the JSON string log entry?

Would make it easier to grasp the different expected cases and how to modify the tests in the future, when there is only a single case per test.

Reply:
I thought that since it is testing the deletion, it would be best to keep both cases in the same test, so you could see the two different possibilities

Reply:
That is valid too.